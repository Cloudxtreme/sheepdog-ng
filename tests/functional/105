#!/bin/bash

# Test recovery of static hash ring

. ./common

for i in `seq 0 2`; do
    _start_sheep $i
done

_wait_for_sheep 3

_cluster_format -c 2 -s

$DOG vdi create test 1000M

_kill_sheep 1
_wait_for_sheep_stop 1

$DOG cluster info | _filter_cluster_info

$DOG node list

_random | $DOG vdi write test &

sleep 1

_start_sheep 1
_wait_for_sheep_recovery 0

$DOG vdi check test

wait
_node_info
