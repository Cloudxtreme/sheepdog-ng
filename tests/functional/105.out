QA output created by 105
using backend plain store
Cluster status: running, auto-recovery disabled

Cluster created at DATE

Epoch Time           Version
DATE      1 [127.0.0.1:7000, 127.0.0.1:7001, 127.0.0.1:7002]
  Id   Host:Port         V-Nodes       Zone
   0   127.0.0.1:7000      	128          0
   1   127.0.0.1:7001      	128          1
   2   127.0.0.1:7002      	128          2

Offline: [127.0.0.1:7001, ]
finish check&repair test
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	153	0	0	0	0	0
1	179	1	0	0	0	0
2	168	1	0	0	0	0
